utils/select.js

```diff
@@ -0,0 +1,20 @@
+export function onClickOption(option, e) {
+  if (!this.$attrs.multiple) {
+    return;
+  }
+
+  const getValue = opt => (this.optionKey ? this.get(opt, this.optionKey) : this.getOptionLabel(opt));
+  const optionValue = getValue(option);
+  const optionIndex = this.value.findIndex(option => getValue(option) === optionValue);
+
+  if (optionIndex < 0) {
+    return;
+  }
+
+  this.value.splice(optionIndex, 1);
+
+  this.$emit('input', this.value);
+  e.preventDefault();
+  e.stopPropagation();
+  this.$refs['select-input'].closeSearchOptions();

```

Comment:
I think checking `attrs.closeOnSelect` would be a good choice here since you're also checking multiple on line two.

Reply:
Makes sense. I'll do it.